src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -186,6 +186,10 @@ private function parseDefinition($id, $service, $file)
                 $parameters[$name] = SimpleXMLElement::phpize($value);
             }
 
+            if ('' === trim($tag['name'])) {
```

Comment:
IMO the trim should not be done and is also not tested.
Since we do not trim the tag when it is added, it should also not validate with trimming. Seems inconsistent.